index.js

```diff
@@ -401,26 +390,38 @@ async function handleSensorRequest(currentLocationId){
       let currentSession = await db.updateSessionState(latestSession.sessionid, currentState, currentLocationId); //Adds the closing state to session
 
       if(await db.closeSession(currentLocationId)){ // Adds the end_time to the latest open session from the LocationID
-        console.log(`Session at ${currentLocationId} was closed successfully.`);
+        helpers.log(`Session at ${currentLocationId} was closed successfully.`);
         start_times[currentLocationId] = null; // Stops the session timer for this location ID
       }
       else{
-        console.log(`Attempted to close session but no open session was found for ${currentLocationId}`);
+        helpers.log(`Attempted to close session but no open session was found for ${currentLocationId}`);
       }
     }
 
-    else if(CHATBOTSTARTSTATES.includes(currentState)) {
+    else if (CHATBOTSTARTSTATES.includes(currentState)) {
         let latestSession = await db.getMostRecentSession(currentLocationId);
 
-        if(latestSession.od_flag == 1) {
-            if(latestSession.chatbot_state == null) {
-              sendInitialChatbotMessage(latestSession);
+        if (latestSession.od_flag === 1) {
+            if (latestSession.chatbot_state === null) {
+                const alertInfo = {
+                    sessionId: latestSession.sessionid,
+                    toPhoneNumber: location.phonenumber,
+                    fromPhoneNumber: location.twilio_number,
+                    message: `This is a ${latestSession.alert_reason} alert. Please check on the bathroom. Please respond with 'ok' once you have checked on it.`,
```

Comment:
Add `${location.location_human} ` in this message to be more clear. Feature request from Sampath :smile:

Reply:
Done in https://github.com/bravetechnologycoop/ODetect-Backend-Local/pull/57